<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFlightPlan</class>
 <widget class="QDialog" name="DialogFlightPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>610</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>610</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{	
	color: #ff0000;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditRoute">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>460</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
}

QScrollBar:vertical
{
	background: #000000;
	width: 5px;
	border: none;
	margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #323232;
}

QScrollBar::add-page:vertical
{
	background: #1e1e1e;
	border: none;
}

QScrollBar::sub-page:vertical
{
	background: #1e1e1e;
	border: none;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFillAll">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fill All</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCallsign_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fligth Rules:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetSquawk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Squawk</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAlternate">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 13px;
}

QLineEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAltitude">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 13px;
}

QLineEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTAS">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 13px;
}

QLineEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTAS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TAS:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCallsign">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Callsign:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDestination">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewValidator">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>420</width>
     <height>175</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView
{;
	color: #c8c8c8;;
	background-color: #000000;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 11px;
	outline: none;
}

QTableView::item:hover
{
    background-color: #4b4b4b;
}

QTableView::item:selected
{

}

QTableView::item:selected:active
{

}

QTableView::item:alternate
{

}

QTableView::item:focus
{
	outline: none;
}


QScrollBar:vertical
{
	background: #000000;
	width: 2px;
	border: none;
	margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #323232;
}

QScrollBar::add-page:vertical
{
	background: #000000;
	border: none;
}

QScrollBar::sub-page:vertical
{
	background: #000000;
	border: none;
}</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditFuelTime">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	selection-background-color: #060064;
	font: 13px;
}

QTimeEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetRoute">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Route</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAltitude">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDeparture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Departure:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetTAS">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set TAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnrTime">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enroute Time:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAcftType">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 13px;
}

QComboBox QAbstractItemView
{	
	color: #c8c8c8;
	background-color: #1e1e1e;
    selection-background-color: #060064;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAlternate">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Alternate:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCallsign">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 13px;
}

QLineEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSquawk">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Squawk:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDeparture">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 13px;
}

QLineEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFlightRules">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 13px;
}

QComboBox QAbstractItemView
{	
	color: #c8c8c8;
	background-color: #1e1e1e;
    selection-background-color: #060064;
	font: 13px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>IFR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VFR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SVFR</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditDestination">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 13px;
}

QLineEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSquawk">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	font: 13px;
}

QLineEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelRoute">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Route:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetCallsign">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Callsign</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFuelTime">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fuel Time:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDepTime">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dep. Time:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditEnrTime">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	selection-background-color: #060064;
	font: 13px;
}

QTimeEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditDepTime">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit:enabled
{
	color: #c8c8c8;
	background-color: #1e1e1e;
	border-style: inset;
	border-width: 2px;
	border-color: #141414;
	selection-background-color: #060064;
	font: 13px;
}

QTimeEdit
{
	color: #323232;
	background-color: #050505;
	border-style: inset;
	border-width: 2px;
	border-color: #0a0a0a;
	font: 13px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAcftType">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Acft Type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
